lib/subjects/remover.rb

```diff
@@ -0,0 +1,62 @@
+module Subjects
```

Comment:
Nitpick: I personally prefer things that are overly domain-specific and/or access AR models not to be in `lib/`. I prefer to keep it leaning towards stuff we could potentially extract to gems. Do you have a stance on `lib/` contents?